src/components/Layout/RightPaneLayout/RightPaneLayout.tsx

```diff
@@ -0,0 +1,52 @@
+import { AnimatePresence, motion } from "framer-motion";
+import React, { ReactElement } from "react";
+import { Css, Palette, px } from "../../../Css";
+import { useRightPaneContext } from "./RightPaneContext";
+
+export function RightPaneLayout({ children, paneBgColor }: { children: ReactElement; paneBgColor?: Palette }) {
+  const { isRightPaneOpen, rightPaneContent } = useRightPaneContext();
+
+  // Todo pass as prop?
+  const paneWidth = 450;
+
+  return (
+    <div css={Css.h100.df.$}>
+      <>
+        <motion.div
+          layout="position"
+          key="rightPaneLayoutPageContent"
+          css={Css.h100.if(!!isRightPaneOpen).overflowX("scroll").mr3.if(!isRightPaneOpen).overflowX("unset").$}
```

Comment:
Nit: could use `overflowXScroll`; granted, we don't have `overflowXUnset`.

Although, do you need `overflowX("unset")`? I think if `isRightPaneOption` is false, then `overflowX("scroll")` just won't be output, so you won't need to unset it?